tests/cocotb/test_snitch_cc.py

```diff
@@ -0,0 +1,125 @@
+# ---------------------------------
+# Copyright 2023 KULeuven
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+# Author: Ryan Antonio (ryan.antonio@esat.kuleuven.be)
+# ---------------------------------
+
+
+# -----------------------------------
+# Imports
+# -----------------------------------
+import subprocess
+import cocotb
+from cocotb.triggers import RisingEdge
+from cocotb.clock import Clock
+from cocotb_test.simulator import run
+import pytest
+
+
+# Reconfigurable parameters
+CLOCK_CYCLES = 20
+
+
+@cocotb.test()
+async def snitch_cc_dut(dut):
+    # Initialize clock
+    clock = Clock(dut.clk_i, 10, units="ns")
+    cocotb.start_soon(clock.start())
+
+    # Reset or intial values
+    dut.rst_ni.value = 0
+
+    # Wait 1 cycle to reset
+    await RisingEdge(dut.clk_i)
+
+    # Deassert reset
+    dut.rst_ni.value = 1
+
+    for i in range(CLOCK_CYCLES):
+        # Simple check if instructions are running normally
+        # instruction_value = hex(dut.i_snitch_cc.i_snitch.inst_data_i.value)
+        # instruction_address = int(dut.instruction_addr_offset.value)
+
+        # cocotb.log.info('---------- Instruction Info ----------')
+        # cocotb.log.info(f'Instruction Value: {instruction_value}')
+        # cocotb.log.info(f'Instruction Addr: {instruction_address}')
+
```

Comment:
This seems useful to have? Should we uncomment?
Can you add asserts for automatic tests >>> manual tests?

Reply:
This test was from a previous one I have in a separate repo. This will be added later so I just commented it first to remind me!

Reply:
Although I think I will do it in the next PR since I need to add the synthetic memories for "testing".